docs/sources/enterprise/activate-license.md

```diff
@@ -55,3 +76,8 @@ Or with an environment variable:
 ```
 GF_SERVER_ROOT_URL=https://grafana.blah.com/
 ```
+
+## Step 4. Restart Grafana
+
+To finalize the installation of Grafana Enterprise, restart Grafana to
```

Comment:
I suggest adding the command(s) to do that in this step.

Reply:
That's makes sense. What systems should we at least target when writing that instruction?

I can add `sudo systemctl restart grafana-server` as the baseline command for people who have followed the most basic installing Grafana as a service on one of the Linux distributions we provide packages for, but when we start coming into Docker, Kubernetes, Windows, macOS, and whatnot territory that'd be a topic that should be covered in its own right in a separate PR by someone who's less of a wannabe system administrator than I am.

Reply:
I would duplicate what we have in the installation instructions. 

Do you really think there are enough different ways to restart Grafana that they deserve a topic? If so, then I will add it to our backlog.

Reply:
I couldn't find instructions for restarting in the installation instructions or anywhere in our docs, but I'll squeeze in a `systemctl` reference so that the user has _some_ level of guidance here. Thanks for pointing it out, I remember when being new and getting a "do X" instruction and being so frustrated that barely anyone would bother sharing even the slightest hint to where to look for instructions on how to do X, and here I'm just as guilty of making the same assumptions about people's skill levels 😅

As for if restart instructions deserve a topic on their own, maybe. We support at least Windows, macOS, Linux, and Docker. The process to install, set up as service, configure, upgrade, restart, and uninstall is different for each of those, and it'd be convenient to be able to refer to a central documentation where the user can see OS specific instructions.

If we decide that we can document for say Linux and then ask readers to translate that for them, I'd be ok with that too. To me, what to target when documenting is a design decision that I'd defer to you.

I don't know enough to provide details for Windows without help, though — and I think it'd be useful to document these key idiosyncrasies by having more comprehensive Windows, Mac, Linux (we list a lot of Linux specifics already, but not in a "Linux overview" page style), and Docker guides covering system operations beyond just installations.

tl;dr: Yes. I do. What I'm not sure about is what priority it should have.